app/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,8 @@
     <string name="rb_mobile">mobile</string>
     <string name="rb_email">email</string>
     <string name="verification_mode">Verification Mode</string>
+    <string name="confirmPassword">Confirm Password</string>
```

Comment:
@Rish02Sharma please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look

Reply:
Done! :+1: